modules/system/classes/MixAssets.php

```diff
@@ -59,8 +69,9 @@ public function init()
         *   public function registerMixPackages()
         *   {
         *       return [
-        *           'package-name-1' => 'winter-mix.js',
+        *           'package-name-1' => 'winter.mix.js',
         *           'package-name-2' => 'assets/js/build.js',
+        *           'package-name-3' => 'assets/css', // winter.mix.js is assumed to be the config file in this path
```

Comment:
@LukeTowers per [your comment](https://github.com/wintercms/winter/pull/444/files#r805385923) here regarding not assuming a file-less path to be pointing to the `winter.mix.js` file, we'll need to remove this example.

```suggestion
```